modules/likes/jetpack-likes-settings.php

```diff
@@ -43,7 +43,7 @@ public function add_meta_box() {
 		 */
 		$title = apply_filters( 'likes_meta_box_title', __( 'Likes', 'jetpack' ) );
 		foreach( $post_types as $post_type ) {
-			add_meta_box( 'likes_meta', $title, array( $this, 'meta_box_content' ), $post_type, 'side', 'default' );
+			add_meta_box( 'likes_meta', $title, array( $this, 'meta_box_content' ), $post_type, 'side', 'default', array( '__back_compat_meta_box' => true ) );
```

Comment:
Note that this is disabling the default metabox even if the Jetpack beta blocks are disabled. We should check if the `likes` extension is available before disabling the old metaboxes.